templates/_linux.yaml

```diff
@@ -32,6 +32,10 @@ objects:
     template:
       spec:
         domain:
+{%- if item.iothreads %} {{''}}
```

Comment:
Why did you keep the {{''}} part? I do not quite understand its purpose here..

Reply:
Just run playbook without {{''}} and check - the template will be wrong. {{''}} helps to add additional \n in case if iothreads exist. 

Checked in the ansible 2.7.0